packages/web/src/components/collection/desktop/PlayButton.tsx

```diff
@@ -0,0 +1,36 @@
+import { MouseEventHandler } from 'react'
+
+import { Button, ButtonType, IconPause, IconPlay } from '@audius/stems'
+import cn from 'classnames'
+
+import styles from './CollectionHeader.module.css'
+import { BUTTON_COLLAPSE_WIDTHS } from './utils'
+
+const messages = {
+  play: 'Play',
+  pause: 'PAuse'
```

Comment:
PAuse

Reply:
probably not a big deal if we transform to uppercase anyway

Reply:
great catch